asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
Thank you for the comment.

The research pattern is always 'space-foo-space'. It's not a check in comments only ,but at least, many identifiers are not counted.  


  
May I take this opportunity to have details about the correct spelling of some terms ?
Frequence of use doesn't imply correctness. Agreed, but what are the correct forms ?  

- 'tailcall'  : so this form is wrong and must be replaced by 'tail call' or 'tail-call'. Damn !  

Some real examples : 
The family of words using 's' or 'z':

- 'initialize' - 'optimization' ... seem correct but are they ?

and what about 

- 'analyse/analyze' - 'specialised/specialized' - 'finalise/finalize' (where 's' is currently the majority) ?

- Also : 'signaled/signalled' - 'labeled/labelled'
- And : 'parentheses/parenthesis' ( a fierce battle)

(and all stuff I forgot ...)

**Note:** I wanted to separate the "real" typos (occurence,wether ...) from the "false" (the others !) but ,finally, I did not do it .
Now, I don't know which solution is the best. A suggestion ? Thank you .